k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_cluster.tpl

```diff
@@ -0,0 +1,86 @@
+{{- define "storageset.node" -}}
+- name: {{ .name }}
+  id: {{ default uuidv4 .id | replace "-" "" | quote }}
+  hostname: {{ .name }}
+  type: {{ .type }}
+{{- end -}}
+
+{{- define "cluster.yaml" -}}
+cluster:
+  name: {{ .Values.clusterName }}
+  id: {{ default uuidv4 .Values.clusterId | replace "-" "" | quote }}
+  node_types:
+  - name: data_node
+    components:
+      - name: utils
+      - name: motr
+        services:
+          - io
+      - name: hare
+    {{- with .Values.clusterStorageVolumes }}
+    storage:
+    {{- range $key, $val := . }}
+    - name: {{ $key }}
+      type: {{ $val.type }}
+      devices:
+        metadata: {{- toYaml $val.metadataDevices | nindent 10 }}
```

Comment:
Minor difference here from before, this creates an array instead of a scalar. The former is the format described in the spec. It appears to work the same as before.